### DISCUSSION TITLE
Practicas FrontEnd

### DISCUSSION THREAD

**Discussion by EsthebanDiego** [Launch-X-Latam/MisionFrontEnd#206] [General]
Hola comparto lo que llevo hasta el momento de prácticas, espero estar haciéndolo bien, estoy tratando de hacer la practica en su mayoría en inglés para también practicarlo un poco,
hasta este momento he utilizado:
- Figma
- Excalidraw
- Miro

[REPOSITORIO](https://github.com/EsthebanDiego/LaunchX-practicas.git)

**Comment by doguedogue** [comment]
genial 🚀🚀🚀 felicidades

**Comment by maxkario** [comment]
Excelente. Hace algunas semanas descubrí Excalidraw y con el hace algunas cosas para la practiva. Pero te quedo genial.
